ui/src/components/Tasks/Widgets/NumericWithUnitWidget/NumericWithUnitWidget.module.css

```
@@ -0,0 +1,20 @@
+/* wrapper around the <input> */
+.wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+/* the little unit badge */
+.unit {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  /* sit right before the spin buttons */
+  right: 2.25rem;
+  white-space: nowrap;
+  font-size: 0.875rem;
+  color: var(--bs-secondary);
+  pointer-events: none; /* don’t block clicks on the input */
```

Best to avoid special characters :wink: 

```suggestion
  pointer-events: none; /* don't block clicks on the input */
```

---

Sorry for these. I did not take a good enough look at these